example/spec/parallel/testAPI/snsRuleSpec.js

```diff
@@ -267,13 +267,18 @@ describe('The SNS-type rule', () => {
     beforeAll(async () => {
       if (beforeAllFailed) return;
       try {
+        const updateParams = {
+          ...updatedRule,
+          state: 'ENABLED',
+          rule: {
+            value: updatedRule.rule.value,
+            type: 'sns',
+          },
```

Comment:
Since we don't have to set `rule.arn` explicitly, we probably don't need to update the `rule` property at all?